js/ionic.md

```diff
@@ -140,8 +138,6 @@ import { AmplifyService }  from 'aws-amplify-angular';
   // ...
 constructor( public amplifyService: AmplifyService ) {
 
-    this.amplifyService = amplifyService;
```

Comment:
we probably should update this to be a private injection and reference it plainly here

Reply:
I doubt want to change the visibility of the variable, and I actually thought it should start public so not to break different projects unexpectedly. I'll change it to private